### ISSUE TITLE
Mobile card not showing slider if more capabilities are used

### ISSUE THREAD

**Issue by kasteleman** [athombv/homey#842]
**What is your Homey version (Settings → System)?**

0.9.3

**What did you try to achieve?**

I try to use the mobile card in a zwave app (com.qubino) in order to show the additional temperature sensor.

_Please include detailed steps how we can reproduce the issue_

This is a part off the code used in the app:

```
  "class": "socket",
  "capabilities": [ "onoff", "dim", "measure_temperature", "measure_power" ],
  "mobile": {
  "components": [
    {
        "id": "icon",
        "capabilities": [ "onoff" ]
    },
    {
        "id": "slider",
        "capabilities": [ "dim" ]
    },
    {
        "id": "sensor",
        "capabilities": [ "measure_temperature" ]
    },
    {
        "id": "sensor",
        "capabilities": [ "measure_power" ]
    }
  ]
  },
  "images": {
      "large": "/drivers/ZMNHDA2/assets/images/large.png",
      "small": "/drivers/ZMNHDA2/assets/images/small.png"
      },
  "settings": [
```

........

**What was the result?**

The measure_power sensor isn't displayed correct in the device card within Chrome (no slider present) and in IOS app not visible at all.

...

**What did you expect as result?**

Displayed correct in both device cards

...

**Any other remarks**

https://files.slack.com/files-pri/T04E9KJ7T-F27C5TGTY/pasted_image_at_2016_09_01_19_18.png
![pasted image at 2016_09_01 19_18](https://cloud.githubusercontent.com/assets/5923368/18180131/c730ccbc-7085-11e6-83ab-4d990467ff93.png)
![slack for ios upload](https://cloud.githubusercontent.com/assets/5923368/18180134/cca48efe-7085-11e6-99ec-80f61d69858d.jpg)

Same for the Fibaro Motion Sensor
...

**Comment by nlrb** [comment]
Looks related to https://github.com/athombv/homey/issues/795.

**Comment by jeroenvollenbrock** [comment]
Can you confirm whether the sensor part of the mobile card is scrollable (even though there is no scroll bar)?

**Comment by kasteleman** [comment]
Just checked again: In both cases i can not scroll

**Comment by Inversion-NL** [comment]
Got the same issue on the SqueezeBox driver I'm currently writing. If it doesn't fit there is no scrollbar below the mobile card items.
The last item (in my case) isn't shown.
If I swap the last two items, the other items isn't shown proving my code and app.json is correct.

**Comment by RobinBol** [comment]
It seems that this issue does not appear in 0.8.39, ran into this issue with another app as well. On 0.8.39 the scrollbar was present, on 0.9.3 not.

**Comment by WeeJeWel** [comment]
Please use 1 sensor component with multiple capabilities, that will fix your issue :)

**Comment by kasteleman** [comment]
No it doesn't. After that my dimmer slider isn't usable.
Informed Robin about that

Regards,

Martin

Op dinsdag 1 november 2016 heeft Emile Nijssen <notifications@github.com>
het volgende geschreven:

**Comment by WeeJeWel** [comment]
How do you mean 'not usable'?

**Comment by kasteleman** [comment]
Send you a screenshot on slack. Slider becomes that thin, you can not use
it.

Op dinsdag 1 november 2016 heeft Emile Nijssen <notifications@github.com>
het volgende geschreven:

**Comment by WeeJeWel** [comment]
Fixed in 0.10.8

**Comment by kasteleman** [comment]
Installed firmware 0.10.9 and that fixed it for me.
